### DISCUSSION TITLE
Single sourcing with Gitbook?

### DISCUSSION THREAD

**Discussion by kfc2023** [GitbookIO/community#266] [0. General bugs and questions] [answered]
### Ask a question, start a conversation, or make an annoucement

Has anyone come up with a clever way to use gitbook along with a single-sourcing strategy? I need to produce two different flavors of a manual for two different audiences. Some of the content will be the same.

**Comment by addisonschultz** [comment, marked as answer]
Hey @kfc2023! Unfortunately right now we don't have a way to re-use content, but it's definitely something on our radar. 

We do though have a way to have two different "types" of a manual/published site in 1 place! You can check out collections to learn more: https://docs.gitbook.com/content-creation/content-structure/what-is-a-collection
